### PR TITLE
Vaporize Bull

### PR DESCRIPTION
**Pull request by Scarlet-Oduko**
## About The Pull Request
Completely removes bull and associated code, sending it into the void. Purging the foul scum!
## Why It's Good For The Game
Bull is a dead end Xeno, causing countless upsets and maddening conversations. Its kit at the end of the day is either too weak or too broken in the case of its primo. It would require a full rework as changing its primo would ultimately make kill bull unless done insanely right. Its not worth the effort to take care of this caste to make sure its not a problem. As well as completely removing it frees up more slots and 'space' for newer and more active castes that will be properly maintained. It is more beneficial to remove then to keep it in for it to sit unchanged forever in purgatory. Github always remember and it can be brought back if someone truly wishes to put in the effort. as of right now its a net negative to be in the game.

TL;DR: Bull is better off dead then in the game, not worth the sanity or insanity of others to be worked on.
## Changelog
:cl:
del: Completely removes Bull code
balance: Bull no longer exists
/:cl:

### CODE REVIEW

**Comment by TiviPlus** [comment]
Change it
ps consider discussing removals prior so I dont need to instantly close them

**Comment by chizzy376** [comment]
speehdmergher

**Comment by Mrrpip** [comment]
I guess you could say it's....Bullshit?

Alright. I'l show myself out now.
